django_q/brokers/orm.py

```diff
@@ -56,13 +57,23 @@ def info(self) -> str:
     def fail(self, task_id):
         self.delete(task_id)
 
-    def enqueue(self, task):
+    def _enqueue_attempt(self, task):
         # list_key might be null (e.g. in a test setup) but OrmQ.key has not-null constraint
         package = self.get_connection().create(
             key=self.list_key or Conf.CLUSTER_NAME, payload=task, lock=timezone.now()
         )
         return package.pk
 
+    def enqueue(self, task):
+        try:
+            return self._enqueue_attempt(task)
```

Comment:
@danleyb2 could you check there is only this enque needs to be called? there is a default retry mechanism and it seems to be handled outside this code block.

Reply:
@jnoortheen Yes i did check and i do understand the django-q code very well from the previous benchmark we did a while back to find the best broker

Reply:
@jnoortheen one thing i wasn't sure is if we should retry more than once

Reply:
I think we can wait on this PR to see if we encounter the same issue again. What do you think @marcbelmont .

Reply:
It's true that we have only one instance of the error. I'm fine either way.